app/controllers/user_confirmations_controller.rb

```diff
@@ -45,8 +45,9 @@ def after_confirmation_path_for(resource)
       end
 
     if resource.reset_password_token.present?
```

Comment:
I think that this should double-check the email confirmation.
```suggestion
    if resource.errors.empty? && resource.reset_password_token.present?
```

Fortunately, the password reset controller doesn't allow resets for unconfirmed email addresses. Otherwise we would have a security hole here. But we shouldn't rely on the password reset controller here and only provide a valid password reset link if the email as been verified correctly.

Reply:
Thanks for reviewing the PR Maikel!
I used github to commit your suggestion :ok_hand:

Reply:
I then changed it to results == 'confirmed'
It makes the method a lot easier to read/understand.